client/js/genre.js

```diff
@@ -1,24 +1,24 @@
 /**
- * fetches youtube api calls from YoutubeServlet.java
+ * @file fetches youtube api calls from YoutubeServlet.java
  * and displays on youtube-genre.html
  */
 
-const genreBlock = document.getElementById('genres');
+document.body.onload = fetchMusicGenre();
```

Comment:
Why are you calling `fetchMusicGenre()` as an onload handler if you call it on line 24?

Reply:
O good point